### DISCUSSION TITLE
How can I remove the folder name from the snippet card, and how can snippets be reordered?

### DISCUSSION THREAD

**Discussion by ShorePatrol** [massCodeIO/massCode#359] [General]
Not sure if this is even possible, but can the folder name be removed from the snippet card? It doesn't need to be there, and takes up too much vertical space.

I wish you could drag and drop the snippets to order them (aside from using the "sort")

Thanks!

**Comment by tzucker02** [comment]
I am relatively new to masscode, but don't see a way to sort snippets, folders, or tags.  Where is the sort ability you speak of?

**Reply by ShorePatrol** [reply]
Under the VIEW menu item, you have "Sort snippets by" - folders can be dragged into whatever order you want.

**Reply by subfighter3** [reply]
there is no such menu under View. The only voice is "Show/Hide Sidebar"…

**Reply by tzucker02** [reply]
That is the only choice I see as well. I am using masscode v4.4 sponsored.

**Reply by tzucker02** [reply]
@ShorePatrol Thanks for the heads up re dragging folders!

**Reply by ShorePatrol** [reply]
Oh wow - there USED to be that option - I just updated to 4.4. sponsored and its now gone - not sure why they would have removed that option.

**Reply by tzucker02** [reply]
@ShorePatrol Which version were you using that had the option? If it was already present, maybe the developer will find it easier to put it back.

**Reply by ShorePatrol** [reply]
I think it was 3.7 - it was 3.something....

**Reply by tzucker02** [reply]
When I started using masscode it was 3.x. I upgraded to sponsored almost immediately and it was 3.x, but I downloaded the 4.x version almost immediately. So I never played around with it at all prior to the 4.x version.
